Clients/src/presentation/components/Modals/FileUpload/index.tsx

```
@@ -0,0 +1,76 @@
+import { ALLOWED_FILE_TYPES, MAX_FILE_SIZE } from "../constants";
+import ErrorModal from "../Error";
+
+const FileUpload: FC = () => {
+  const theme = useTheme();
+  const [file, setFile] = useState<File | null>(null);
+  const [error, setError] = useState<string | null>(null);
+  const [isErrorModalOpen, setIsErrorModalOpen] = useState<boolean>(false);
+
+  const handleFileUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setError(null);
+    if (e.target.files && e.target.files[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)